src/features/options/useGetOptions.ts

```diff
@@ -52,20 +54,32 @@ export interface OptionsResult {
 
 const defaultOptions: IOption[] = [];
 
+type SortOrder = 'asc' | 'desc';
+const compareOptionAlphabetically =
+  (sortOrder: SortOrder = 'asc') =>
+  (a: IOption, b: IOption) => {
+    const comparison = a.label.localeCompare(b.label, ['nb', 'en'], { sensitivity: 'base', numeric: true });
```

Comment:
We should use the current language instead of `['nb', 'en']` here. You cat get the currently selected language key via:

```typescript
const { selectedLanguage } = useLanguage();
```